.gitignore

```
@@ -1,9 +1,9 @@
 /node_modules
 /data
 /config
-
+.env
```

```suggestion
/.env
```

`/frontend/.env` ist bewusst im Repo.